.github/workflows/elixir.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Elixir CI
```

Comment:
What do you think of calling this "CI" instead of "Elixir CI"?

Reply:
I agree with you; "CI" instead of "Elixir CI" is more appropriate.